Sources/StreamChatUI/Components.swift

```diff
@@ -230,7 +233,20 @@ public struct _Components<ExtraData: ExtraDataTypes> {
     /// The `UICollectionViewCell` subclass that shows channel information.
     public var channelCell: _ChatChannelListCollectionViewCell<ExtraData>.Type =
         _ChatChannelListCollectionViewCell<ExtraData>.self
-
+    
+    /// The `UICollectionViewCell` subclass that shows placeholder for channel information.
+    public var channelPlaceholderCell: _ChatChannelListPlaceholderCollectionViewCell<ExtraData>.Type =
+        _ChatChannelListPlaceholderCollectionViewCell<ExtraData>.self
```

Comment:
IMHO this placeholder cell logic will make it harder to customize channel list cells. There are libraries that can create a skeleton from an already existing cell. We should check those libraries and implement a similar logic: create the placeholder cell from an already existing cell.

Reply:
I changed this to adding shimmers  to current cell which has option to change the shimmered views. Also deleted the placeholderCell :)